.github/workflows/automatic-pr-update.yml

```diff
@@ -2,24 +2,16 @@ name: Automatic PR Merger
 
 on:
   push: {} # update PR when base branch is updated
-  status: {} # try to merge when other checks are completed
-  pull_request_review: # try to merge after review
-    types:
-      - submitted
-      - edited
-      - dismissed
-  pull_request: # try to merge if labels have changed (white/black list)
-    types:
-      - labeled
-      - unlabeled
 
 jobs:
   # thats's all. single step is needed - if PR is mergeable according to
   # branch protection rules it will be merged automatically
   mergepal:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: rasahq/merge-pal-action@master
+      - uses: actions/checkout@v1
+      - uses: rasahq/update-pr-branch@601ac3dadf708b14e7f0252d98230538235f99d9
+        if: github.repository == 'RasaHQ/rasa'
         with:
           token: ${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN || secrets.GITHUB_TOKEN}}
+          required_approval_count: 0
```

Comment:
is this a workaround or why do we need it?

Reply:
if it's a workaround then I'd vote to add a comment

Reply:
yes it is - this is an odd requirement from the original plugin. I could have changed it in the forked actions code but I wanted the changes to stay minimal.

Reply:
makes complete sense 👍🏻